From 9e3a611a3c1177878df2f09b6c4e0bee13239ba3 Mon Sep 17 00:00:00 2001
From: Daniel Nicoletti <dantti12@gmail.com>
Date: Tue, 6 May 2014 12:44:51 -0300
Subject: [PATCH 03/12] Remove the metatype registration for Transaction types
 as is't a bit useless since it depends on the namespace being used better
 leave this to the client to register

---
 src/daemon.cpp | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/src/daemon.cpp b/src/daemon.cpp
index 06907f4..fdea8a7 100755
--- a/src/daemon.cpp
+++ b/src/daemon.cpp
@@ -66,25 +66,6 @@ Daemon::Daemon(QObject *parent) :
     if (!db.open()) {
         qDebug() << "Failed to initialize the desktop files database";
     }
-
-    qRegisterMetaType<PackageKit::Daemon::Network>("PackageKit::Daemon::Network");
-    qRegisterMetaType<PackageKit::Daemon::Authorize>("PackageKit::Daemon::Authorize");
-    qRegisterMetaType<PackageKit::Transaction::InternalError>("PackageKit::Transaction::InternalError");
-    qRegisterMetaType<PackageKit::Transaction::Role>("PackageKit::Transaction::Role");
-    qRegisterMetaType<PackageKit::Transaction::Error>("PackageKit::Transaction::Error");
-    qRegisterMetaType<PackageKit::Transaction::Exit>("PackageKit::Transaction::Exit");
-    qRegisterMetaType<PackageKit::Transaction::Filter>("PackageKit::Transaction::Filter");
-    qRegisterMetaType<PackageKit::Transaction::Message>("PackageKit::Transaction::Message");
-    qRegisterMetaType<PackageKit::Transaction::Status>("PackageKit::Transaction::Status");
-    qRegisterMetaType<PackageKit::Transaction::MediaType>("PackageKit::Transaction::MediaType");
-    qRegisterMetaType<PackageKit::Transaction::DistroUpgrade>("PackageKit::Transaction::DistroUpgrade");
-    qRegisterMetaType<PackageKit::Transaction::TransactionFlag>("PackageKit::Transaction::TransactionFlag");
-    qRegisterMetaType<PackageKit::Transaction::TransactionFlags>("PackageKit::Transaction::TransactionFlags");
-    qRegisterMetaType<PackageKit::Transaction::Restart>("PackageKit::Transaction::Restart");
-    qRegisterMetaType<PackageKit::Transaction::UpdateState>("PackageKit::Transaction::UpdateState");
-    qRegisterMetaType<PackageKit::Transaction::Group>("PackageKit::Transaction::Group");
-    qRegisterMetaType<PackageKit::Transaction::Info>("PackageKit::Transaction::Info");
-    qRegisterMetaType<PackageKit::Transaction::SigType>("PackageKit::Transaction::SigType");
 }
 
 void DaemonPrivate::setupSignal(const QString &signal, bool connect)
-- 
2.0.1

